src/github.com/getlantern/flashlight/main/pac.go

```diff
@@ -76,31 +76,27 @@ func genPACFile() {
 	}
 	formatter :=
 		`var bypassDomains = %s;
-		function FindProxyForURL(url, host) {
-			if (isPlainHostName(host) // including localhost
-			|| shExpMatch(host, "*.local")) {
+	function FindProxyForURL(url, host) {
+		if (isPlainHostName(host) // including localhost
+		|| shExpMatch(host, "*.local")) {
+			return "DIRECT";
+		}
+		// only checks plain IP addresses to avoid leaking domain name
+		if (/^[0-9.]+$/.test(host)) {
+			if (isInNet(host, "10.0.0.0", "255.0.0.0") ||
+			isInNet(host, "172.16.0.0",  "255.240.0.0") ||
+			isInNet(host, "192.168.0.0",  "255.255.0.0") ||
+			isInNet(host, "127.0.0.0", "255.255.255.0")) {
 				return "DIRECT";
 			}
-			// only checks plain IP addresses to avoid leaking domain name
-			if (/^[0-9.]+$/.test(host)) {
-				if (isInNet(host, "10.0.0.0", "255.0.0.0") ||
-				isInNet(host, "172.16.0.0",  "255.240.0.0") ||
-				isInNet(host, "192.168.0.0",  "255.255.0.0") ||
-				isInNet(host, "127.0.0.0", "255.255.255.0")) {
-					return "DIRECT";
-				}
-			}
-			// Lantern desktop version proxies only http and https
-			if (url.substring(0, 4) != 'http') {
```

Comment:
It's a confusing diff, but it's just whitespace changes.  I'm tempted to stick with this version since it worked in Edge and other browsers.